src/components/prediction/ResultTable.js

```diff
@@ -0,0 +1,83 @@
+import React, {PureComponent} from 'react';
+import {DataTable, TableBody, TableColumn, TableHeader, TablePagination, TableRow} from 'react-md/lib/DataTables/index';
+import PropTypes from 'prop-types';
+import {jobRunPropType} from '../../propTypes';
+import JsonHolder from '../validation/JsonHolder';
+
+/* eslint-disable camelcase */
+class ResultTable extends PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {slicedData: this.props.jobs.slice(0, 10)};
+  }
+
+  handlePagination(start, rowsPerPage) {
+    this.setState({slicedData: this.props.jobs.slice(start, start + rowsPerPage)});
+  }
+
+  componentDidUpdate(prevProps) {
+    if (prevProps.jobs.length !== this.props.jobs.length) {
+      this.setState({slicedData: this.props.jobs.slice(0, 10)});
+    }
+  }
+
+  componentDidMount() {
+    const intervalId = setInterval(() => {
+      this.props.onRequestJobs();
+    }, 10000);
+    this.setState({intervalId: intervalId});
+  }
+
+  componentWillUnmount() {
+    clearInterval(this.state.intervalId);
+  }
+
+  getHeaderColumns() {
+    const headers =
+      ['id', 'Status', 'Type', 'Created date', 'Modified date', 'Split', 'Error', 'Results', 'Configuration'];
+
+    return headers.map((header) => {
+        let grow = false;
+        if (header === 'Configuration') {
+          grow = true;
+        }
+        return <TableColumn key={header} grow={grow}> {header}</TableColumn>;
+      }
+    );
+  }
+
+  render() {
+    return (<DataTable baseId="simple-pagination" plain>
+      <TableHeader>
+        <TableRow selectable={false}>
+          {this.getHeaderColumns()}
+        </TableRow>
+      </TableHeader>
+      <TableBody>
+        {this.state.slicedData.map(
+          ({id, type, status, created_date, modified_date, splitName, config, error, result}) => (
+            <TableRow key={id} selectable={false}>
+              <TableColumn numeric>{id}</TableColumn>
+              <TableColumn>{status}</TableColumn>
+              <TableColumn>{type}</TableColumn>
+              <TableColumn>{new Date(created_date).toLocaleString()}</TableColumn>
+              <TableColumn>{new Date(modified_date).toLocaleString()}</TableColumn>
+              <TableColumn>{splitName}</TableColumn>
+              <TableColumn>{error}</TableColumn>
+              <TableColumn>{JSON.stringify(result)}</TableColumn>
```

Comment:
instead of stringify there is a component to present JSON data

Reply:
that you are using in the next line. better to have it consistent. but i don't know how it looks like as there are no storybook things or tests with sample data

Reply:
Sorry but I didn't understand how to use storybook and I never been used to make tests, now I'll fix everything when I understand what I need to pull and where i need to push